e2e/go.mod

```diff
@@ -1,21 +1,21 @@
 module sigs.k8s.io/cluster-api-provider-kubevirt/e2e
 
-go 1.20
+go 1.22.0
 
 require (
 	github.com/golang/glog v1.0.0
-	github.com/onsi/ginkgo/v2 v2.13.0
-	github.com/onsi/gomega v1.28.0
+	github.com/onsi/ginkgo/v2 v2.17.1
+	github.com/onsi/gomega v1.32.0
```

Comment:
* ginkgo latest version is v1.18.0
* gomega latest version is v1.33.1

Could you please set to the latest versions?